pkg/chunk/aws/s3_storage_client_test.go

```diff
@@ -0,0 +1,107 @@
+package aws
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"net/url"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/stretchr/testify/require"
+
+	"github.com/cortexproject/cortex/integration/e2e"
+	e2edb "github.com/cortexproject/cortex/integration/e2e/db"
+	"github.com/cortexproject/cortex/pkg/util/flagext"
+)
+
+func TestS3Client(t *testing.T) {
+	networkName := "s3-test"
+	bucketName := "test"
+
+	s, err := e2e.NewScenario(networkName)
```

Comment:
This test case should be in the `integrations/` directory. Otherwise the fixtures it relies on won't be configured correctly.